rector.php.tpl

```diff
@@ -1,18 +1,32 @@
 <?php
 // rexfactor auto generated file - do not edit, delete, rename
 
+use Rector\CodingStyle\Rector\FuncCall\ConsistentPregDelimiterRector;
 use Rector\Config\RectorConfig;
 use Rector\Set\ValueObject\SetList;
 use Rector\PHPUnit\Set\PHPUnitSetList;
+use Rector\Php80\Rector\Identical\StrEndsWithRector;
+use Rector\Php80\Rector\Identical\StrStartsWithRector;
+use Rector\Php80\Rector\NotIdentical\StrContainsRector;
 
 return static function (RectorConfig $rectorConfig): void {
     $rectorConfig->sets([
         %%RECTOR_SETS%%
     ]);
 
+    // redaxo core ships with polyfills, so we can always apply these rectors
+    $rectorConfig->rules([
+        StrEndsWithRector::class,
+        StrStartsWithRector::class,
+        StrContainsRector::class
+    ]);
```

Comment:
depends on https://github.com/rectorphp/rector/issues/7867

Reply:
I will enforce it via redaxo code style instead of rector rules, which works out of the box for me

https://github.com/redaxo/php-cs-fixer-config/issues/5